scripts/src/main/resources/scripts/environment-project.bat

```diff
@@ -79,10 +79,11 @@ rem %~1: variable name
 rem %~2: variable value (may contain ~ or ${var})
 :set_variable
 setlocal EnableDelayedExpansion
-set "value=%~2"
+set "value=%~2 %~3"
 rem replace ${var} variable syntax with windows %var% syntax
-set "value=!value:${=%%!
-set "value=!value:}=%%!
+set value=!value:${=%%!
+set value=!value:}=%%!
+set value=!value:"=%!
```

Comment:
great. So this line replaces double quotes with nothing (empty string) if I get it right.

Reply:
Yes